app/src/main/res/layout/activity_search.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
any particular reason for using frame layout?

Reply:
Set event details to this frame layout when recycle view item is clicked